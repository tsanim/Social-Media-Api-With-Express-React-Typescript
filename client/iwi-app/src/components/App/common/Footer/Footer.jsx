import React from 'react';

function Footer(props) {
    return (
        <footer>
            <p>
                &#9400; CopyRights Tsani Mazalov 2019
            </p>
        </footer>
    )
}

export default Footer;