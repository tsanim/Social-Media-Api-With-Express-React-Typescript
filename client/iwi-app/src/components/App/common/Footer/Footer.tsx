import React from 'react';

function Footer() {
    return (
        <footer>
            <p>
                &#9400; CopyRights Tsani Mazalov 2019
            </p>
        </footer>
    )
}

export default Footer;