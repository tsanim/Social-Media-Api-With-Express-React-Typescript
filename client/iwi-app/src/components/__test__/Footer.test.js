import React from 'react';
import Footer from '../App/common/Footer/Footer';
import { shallow } from 'enzyme';

describe('Footer component', () => {
    it('should render initial layout', () => {
        const enzymeWrapper = shallow(<Footer />);

        expect(enzymeWrapper.getElements()).toMatchSnapshot();
    });

    it('should render child p tag', () => {
        const enzymeWrapper = shallow(<Footer />);

        expect(enzymeWrapper.find('p')).toHaveLength(1);
    });
});