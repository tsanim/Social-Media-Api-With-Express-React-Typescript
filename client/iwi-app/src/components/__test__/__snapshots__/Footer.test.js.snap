// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render initial layout 1`] = `
Array [
  <footer>
    <p>
      â’¸ CopyRights Tsani Mazalov 2019
    </p>
  </footer>,
]
`;
